packages/odyssey/src/scss/components/_input-field.scss

```diff
@@ -84,6 +84,24 @@
   }
 }
 
+.ods-fieldset--attached {
+  display: flex;
+  max-width: $max-line-length;
+
+  .ods-button {
+    border-radius: 0 $base-border-radius $base-border-radius 0;
+  }
+
+  .ods-text-input {
+    border-right: 0;
+    border-radius: $base-border-radius 0 0 $base-border-radius;
+
+    &:focus {
+      z-index: 1;
```

Comment:
whats the deal with the z-index here? Does it have something to do with the outline/box shadow?

Reply:
Yeah, this ensures that the focus state isn't hidden under the adjoining button, but only when focused (don't want to mess around with the z-index otherwise).